python/servo/command_base.py

```diff
@@ -888,6 +912,10 @@ def run_cargo_build_like_command(
                 features.append("webgl-backtrace")
             if self.config["build"]["dom-backtrace"]:
                 features.append("dom-backtrace")
+
+            if self.media_stack.is_dummy() and "--no-default-features" not in cargo_args:
+                args += ["--no-default-features"]
```

Comment:
Hrm. There is an issue here and it's that using the dummy media stack disables the "max_log_level", "native-bluetooth", and "webdriver" features. Perhaps we should make these features non-configurable? I'm not sure what the right thing to do here is.

Reply:
Ah, that's why I wanted ports so I could detect which features to enable. Webdriver should probably be hidden behind pref.

We could also move gstreamer enabling into cargo per target, then mach would only need to handle cross and forced dummy cases.
inspiration: https://stackoverflow.com/questions/57972355/enable-a-cargo-feature-by-default-when-the-target-arch-is-wasm